src/SecTester.Repeater/Bus/SocketIoRepeaterBus.cs

```diff
@@ -48,13 +49,13 @@ private void DelegateEvents()
     _connection.On("error", response =>
     {
       var err = response.GetValue<RepeaterError>();
-      ErrorOccurred?.Invoke(new(err.Message));
+      ErrorOccurred?.Invoke(new Exception(err.Message));
     });
 
     _connection.On("update-available", response =>
     {
       var version = response.GetValue<RepeaterVersion>();
-      UpgradeAvailable?.Invoke(new(version.Version));
+      UpgradeAvailable?.Invoke(new Version(version.Version));
```

Comment:
Use shorthand option
```suggestion
      UpgradeAvailable?.Invoke(new(version.Version));
```